### PR TITLE
treehouses tor deleteall (fixes #423)

### LINKED ISSUE #423: treehouses tor deleteall

### PR DESCRIPTION
**Pull request by Kid243**

### CODE REVIEW

File: modules/tor.sh [hunk 1]
```diff
@@ -54,6 +54,32 @@ function tor {
 
     restart_service tor
     echo "Success: the port has been added"
+  elif [ "$1" = "delete" ]; then
```
Comment (dogi) [inline modules/tor.sh]: @Kid243 this is a branch from a branch

Comment (Kid243) [reply modules/tor.sh]: @dogi not a branch from a branch, I copy pasted the code from delete1port into this one anticipating the possibility that delete1port might be merged first

**Comment by Kid243** [comment]
@dogi done with conflicts

File: modules/tor.sh [hunk 2]
```diff
@@ -73,7 +73,14 @@ function tor {
...
     echo "Port $2 has been deleted"
+  elif [ "$1" = "deleteall" ]; then
+    if [ -n "$2" ]; then
+      echo "Error: wrong synthax"
+      exit 1
+    fi
 
+    sed -i "/^HiddenServicePort /d" /etc/tor/torrc
+    echo "All ports have been deleted"
   elif [ "$1" = "stop" ]; then
```
Comment (phamduchongan93) [inline modules/tor.sh]: `restart_service tor`

**Pull request merged**